Packs/CiscoFirepower/Integrations/CiscoFirepower/CiscoFirepower.py

```diff
@@ -1469,120 +3606,844 @@ def get_task_status_command(client: Client, args: Dict) -> Tuple[str, Dict, Dict
+    If no ID is specified, retrieves list of intrusion policies.
+    - GET arguments: intrusion_policy_id, include_count.
+    - LIST arguments: expanded_response, limit, page, page_size.
+
+    Args:
+        client (Client): Session to Cisco Firepower Management Center to run desired requests.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Returns:
+        CommandResults: Information about Intrusion Policies.
+    """
+    # GET arguments
+    intrusion_policy_id = args.get('intrusion_policy_id', '')
+    include_count = argToBoolean(args.get('include_count', 'False'))
+    # LIST arguments
+    limit = arg_to_number(args.get('limit', 0))
+    page = arg_to_number(args.get('page', 0))
+    page_size = arg_to_number(args.get('page_size', 0))
+    expanded_response = argToBoolean(args.get('expanded_response', 'False'))
+
+    is_get_request = check_is_get_request(
+        get_args=[intrusion_policy_id, include_count],
+        list_args=[limit, page, page_size, expanded_response]
+    )
+
+    if is_get_request:
+        raw_response = client.get_intrusion_policy(
+            intrusion_policy_id=intrusion_policy_id,
+            include_count=include_count,
+        )
+
+    else:  # is_list_request
+        raw_response, raw_responses = client.list_intrusion_policy(
+            page=page,
+            page_size=page_size,
+            limit=limit,
+            expanded_response=expanded_response,
+        )
+
+    command_results = get_commandresults(
+        raw_response=raw_response,
+        command_headers_by_keys=INTRUSION_POLICY_HEADERS_BY_KEYS,
+        command_title=f'Fetched {INTRUSION_POLICY_TITLE}',
+        command_context=INTRUSION_POLICY_CONTEXT,
+    )
+
+    if not is_get_request:
+        command_results.raw_response = raw_responses
+
+    return command_results
+
+
+def update_intrusion_policy_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Modifies the Intrusion Policy associated with the specified ID.
+
+    Args:
+        client (Client): Session to Cisco Firepower Management Center to run desired requests.
+        args (Dict[str, Any]): Arguments passed down by the CLI to provide in the HTTP request.
+
+    Raises:
+        ValueError: In case none of the update arguments were entered.
+
+    Returns:
+        CommandResults: Information about the created Intrusion Policy
+    """
+    intrusion_policy_id = args['intrusion_policy_id']
+    name = args.get('name', '')
+    basepolicy_id = args.get('basepolicy_id', '')
+    description = args.get('description')
+    inspection_mode = args.get('inspection_mode')
+    replicate_inspection_mode = arg_to_optional_bool(args.get('replicate_inspection_mode'))
+
+    update_arguments = (name, basepolicy_id, description, inspection_mode)
+
+    if not any(update_arguments):
+        raise ValueError(f'Please enter one of the update arguments: {update_arguments}')
```

Comment:
please mention the field names explicitly, otherwise it will show as `Please enter one of the update arguments: "", "", "", None` or similar

Reply:
Silly me, can't believe I've actually done that. 😅
Fixed.